File: 71A - Way Too Long Words/WTLW.cs
using System;

class WTLW
{
    static void Main()
    {
        int max = int.Parse(Console.ReadLine());

        String[] tempStrings = new String[max];

        for (int i = 0; i < max; i++) {
            tempStrings[i] = Console.ReadLine();
        }

        foreach (String str in tempStrings)
        {
            if (str.Length <= 10) {
                Console.WriteLine(str);
            }
            else {
                Console.WriteLine(str.ElementAt(0) + (str.Length - 2).ToString() + str.ElementAt(str.Length - 1));
            }
        }
    }
}

File: 71A - Way Too Long Words/WTLW.py
max = int(input())

tempStrings = list()

for i in range(max):
    tempStrings.append(input())

for word in tempStrings:
    if (len(word) <= 10):
        print(word)
    else:
        print(word[0] + str(len(word) - 2) + word[-1])